### ISSUE TITLE
[Feature Request] Token Vault - Option to sort entries

### ISSUE THREAD

**Issue by MorgothSauron** [standardnotes/forum#657]
It would be nice to have an option to sort the tokens by service name or account name. In the worst case I can still switch to the plain text editor, copy the content and sort it myself than paste back the new list. But it would be nice if the application could do it natively.

**Comment by mobitar** [comment]
Yeah, I'm sure we'll improve on the editor with features like this in the future.
